tests/utils.py

```diff
@@ -917,7 +917,7 @@ def snapshot_context(token, ignores=None, tracer=None, async_mode=True, variants
         # Wait for the traces to be available
         if wait_for_num_traces is not None:
             traces = []
-            for i in range(20):
+            for i in range(50):
```

Comment:
I don't actually think so, this was a dead end in me investigating the flaky test.

I left the change in as it seems like it would be useful in the future (increase the polling from 2 seconds to 5)